cmd/snap/cmd_quota.go

```diff
@@ -45,24 +42,45 @@ The quotas command shows all quota groups.
 `)
 
 var shortRemoveQuotaHelp = i18n.G("Remove quota group")
-var longRemoveQuotaHelp = i18n.G(`The remove-quota command removes the given quota group.`)
+var longRemoveQuotaHelp = i18n.G(`
+The remove-quota command removes the given quota group.
 
-type cmdQuota struct {
-	clientMixin
-	// MaxMemory and MemoryMax are mutually exclusive and provided for
-	// convienience, they mean the same.
-	MaxMemory  string `long:"max-memory" optional:"true"`
-	MemoryMax  string `long:"memory-max" optional:"true"`
-	Parent     string `long:"parent" optional:"true"`
-	Positional struct {
-		GroupName string              `positional-arg-name:"<group-name>" required:"true"`
-		Snaps     []installedSnapName `positional-arg-name:"<snap>" optional:"true"`
-	} `positional-args:"yes"`
-}
+Quota groups with sub-groups 
+`)
+
+var shortSetQuotaHelp = i18n.G(`Create or update a quota group.`)
+var longSetQuotaHelp = i18n.G(`
+The set-quota command updates or creates a quota group with the specified set of
+snaps.
+
+A quota group sets resource limits (currently maximum memory only) on the set of
+snaps that belong to it. Snaps can be at most in one quota group. Quota groups
+can be nested.
+
+All snaps provided are appended to the group; to remove a snap from a
```

Comment:
I wonder if some of the unsupported things should be prefixed with "currently" or it adds no value as there's no immediate timeline to change this.

Reply:
I think some of them are important to include even if we don't have an immediate timeline to change them, since it's kinda unexpected that you can't do certain things, such as move a group into another group feels an awful lot like just adding a snap to an existing group, but under the hood it is more complicated (and the CLI for that is maybe weird to handle, what about snaps that are named the same as quota groups?) The other thing that's a bit unclear/unexpected is decreasing a limit, it's a bit unexpected that you can increase the limit, but you can't decrease the limit.

that being said I suppose we could just explain the limitations in the docs instead of the help text here.